docs/api.rst

```diff
@@ -137,6 +137,7 @@ Integrals
 
     desc.integrals.Bounce2D
     desc.integrals.Bounce1D
+    desc.integrals.compute_B_plasma
```

Comment:
we might want to hold off on this for a bit, since I think there may be some overlap or potential confusion with #1847. This method only calculates B_plasma on the plasma boundary, while the stuff in #1847 is more general and would likely make a better public method.